src/lib/kwlib.sh

```diff
@@ -4,6 +4,7 @@ include "${KW_LIB_DIR}/lib/kw_db.sh"
 include "${KW_LIB_DIR}/lib/kw_time_and_date.sh"
 
 ENV_DIR='envs'
+TMP_DIR='/dev/shm'
```

Comment:
This name seems too generic... try something like `KW_SHARED_MEMORY_DEFAULT_DIR`... or something shorter but meaningful. Also, add a small doc about this global variable.